deployments/apiv2/services/users/src/main.py

```diff
@@ -3,6 +3,7 @@
 from typing import Union, List, Optional
 import uuid
 from datetime import datetime
+from jobs import check_customer_quota
```

Comment:
Can you run pre-commit checks? This will move back down to local imports

Reply:
I have been running pre commit but it did not catch this, I will change it back manually.